src/ui/units/collections/components/CollectionBatchPanel/CollectionBatchPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import type {ActionsPanelProps} from '@gravity-ui/components';
+import {ActionsPanel} from '@gravity-ui/components';
+import {ArrowRight, TrashBin} from '@gravity-ui/icons';
+import {Icon} from '@gravity-ui/uikit';
+import {I18n} from 'i18n';
+
+export type CollectionBatchAction = 'move' | 'delete';
+
+const i18n = I18n.keyset('collections');
+
+export type BatchPanelProps = {
+    countForMove?: number;
+    countForDelete?: number;
+    onAction: (action: CollectionBatchAction) => void;
+    onClose: () => void;
+    className?: string;
+};
+
+export const CollectionBatchPanel = ({
+    countForMove,
+    countForDelete,
+    className,
+    onAction,
+    onClose,
+}: BatchPanelProps) => {
+    const actions: ActionsPanelProps['actions'] = React.useMemo(
+        () => [
+            {
+                id: 'move',
+                button: {
+                    props: {
```

Comment:
Can we disable the button if count is 0? And maybe we need hide the dropdown item in this case.

Reply:
added hide items when count = 0